aiida_crystal17/validation/schemas/inputd12.schema.json

```
@@ -365,6 +365,17 @@
                 "XXLGRID"
               ]
             },
+            "exact_exchange": {
+              "description": "the (pruned) integration grid",
+              "type": "integer",
+              "minimum": 0
+            },
+            "non_local": {
+              "description": "the (pruned) integration grid",
+              "type": "array",
```

```suggestion
              "type": "array",
              "items": {
                 "type": "integer"
              },
```

is this correct for the item type?

---

This is improved now with descriptions for each item. And these should be `number`.